ir.h

```diff
@@ -374,7 +392,7 @@ typedef struct _ir_insn {
 				union {
 					IR_STRUCT_LOHI(
 						uint8_t        op,
-						uint8_t        type;
+						uint8_t        type
```

Comment:
This ``;`` should probably be added in the ``IR_STRUCT_LOHI`` macro

Reply:
This seems to have been a typo in the original variant. As far as i can see, it's always used with a semicolon at the end, like `IR_STRUCT_LOHI(...);`. It's that the second semicolon after the second member that seems to have made issue with MSVC. 

Thanks

Reply:
Yeah. This is my problem. I can take care after this is merged.